packages/js-runtime/bundler/browser.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
All the js-runtime changes are unneeded, this is what `resolver.ts` (and the EngineResolver in `engine.ts`)  handles for us